styles.css

```diff
@@ -0,0 +1,170 @@
+body {
+    margin: 15;
+    font-family: Arial, Holvetica ;
+    background-image: linear-gradient(to bottom right,#D0F1F5, #D0F1F5,#A3DCE2, #1CD0E4,#52C1CD)
+  }
+  .topnav {
+    overflow: hidden;
+    background-color: #333;
+  }
+  .topnav a {
```

Comment:
The style of the navbar looks very much like a W3 school example. That is ok! But it perhaps would look better if it would be more align with the whole page style, in the same nice color palette :)